### DISCUSSION TITLE
Requesting a BEX extension built on `Quasar CLI with Vite` implementing `manifest version v3` as an example project

### DISCUSSION THREAD

**Discussion by livan2r** [quasarframework/quasar#12935] [CLI - BEX mode] [answered]
I read that Quasar BEX is supporting now manifest version v3 and CLI with Vite. I followed the related documentation but the generated project is running into many issues, so I suppose that I might be doing something wrong. For example, the default Content Script (my-content-script.js) generated by default includes the keyword 'export' but Chome is complaining about that: Uncaught SyntaxError: Unexpected token 'export'
Does anyone know about a tutorial or BEX extension based on `Quasar CLI with Vite` implementing `manifest version v3`, which I could follow or use as an example?

**Comment by rstoenescu** [comment, marked as answer]
You're not installing it in the browser from the correct location. Install it from dist/bex. This is an important difference from the usage in q/app-webpack. It's even highlithed in the terminal on quasar dev command.

**Reply by livan2r** [reply]
Oh, I guess I didn't read that important part. From dist/bex the extension loaded successfully. Thanks a lot.
